### PR TITLE
#271 calculate the date in the tests

### LINKED ISSUE #271: ` test_runs_when_run_duration_ms_is_nil` fails
https://github.com/zerocracy/judges-action/actions/runs/10367594772/job/28699376905

### PR DESCRIPTION
**Pull request by Suban05**
Added stub for `Time.now`
Fixes #271

### CODE REVIEW

**Comment by Suban05** [comment]
@Yegorov please review
I'm not sure about the names

File: test/test__helper.rb [hunk 1]
```diff
@@ -45,4 +45,12 @@ def load_it(judge, fb)
     $loog = Loog::NULL
     load(File.join(__dir__, "../judges/#{judge}/#{judge}.rb"))
   end
+
+  def test_date
```
Comment (Yegorov) [inline test/test__helper.rb:49]: @Suban05 It seems if the method begins with `test_` - this is test case in minitest

File: test/test__helper.rb [hunk 2]
```diff
@@ -45,4 +45,12 @@ def load_it(judge, fb)
...
+    test_time.to_date.to_s
+  end
+
+  def test_time
+    Time.now - (28 * 24 * 60 * 60)
```
Comment (Yegorov) [inline test/test__helper.rb:54]: @Suban05 what is `(28 * 24 * 60 * 60)`? why 28 days?

Comment (Suban05) [reply test/test__helper.rb:54]: @Yegorov see [here](https://github.com/Suban05/fbe/blob/master/lib/fbe/regularly.rb)
Am I right?

Comment (Yegorov) [reply test/test__helper.rb:54]: @Suban05 No, `days` is variable and can have any value different from 28.
For fix this [issue](https://github.com/zerocracy/judges-action/issues/271) use stub for `Time.now`.
See this [example](https://github.com/zerocracy/judges-action/blob/master/test/judges/test-quality-of-service.rb#L279)

File: test/test__helper.rb (hunk 1 above)
Comment (Yegorov) [reply test/test__helper.rb:49]: @Suban05 see  [this comment](https://github.com/zerocracy/judges-action/pull/277#discussion_r1715723449)

File: test/test__helper.rb (hunk 2 above)
Comment (Suban05) [reply test/test__helper.rb:54]: @Yegorov exactly, thanks!

**Comment by Suban05** [comment]
@Yegorov fixed

**Review by Yegorov — APPROVED** [review]
@Suban05 perfectly!

**Comment by Suban05** [comment]
@yegor256 fixed

**Comment by yegor256** [comment]
@Suban05 thanks!

**Comment by 0crat** [comment]
@Yegorov 🎉 Great job on the review! You've earned **15 points**: +15 base, +10 for your 10 insightful comments, with a slight -10 adjustment for 4 hits-of-code. Keep up the excellent work! Your new balance is [294 points](https://zerocracy.github.io/judges-action/zerocracy-vitals.html). 💪

**Pull request merged**